paddle/fluid/framework/tensor_test.cc

```diff
@@ -74,6 +74,22 @@ TEST(Tensor, MutableData) {
     p2 = src_tensor.mutable_data<float>(framework::make_ddim({2, 2}),
                                         platform::CPUPlace());
     EXPECT_EQ(p1, p2);
+
```

Comment:
this for testing the different datatype tensor can be shared. Because all of the reused tensor is output, so it will call the mutable data correctly at the reused op.